### ISSUE TITLE
New translation: Kurdish Central (Sorani)

### ISSUE THREAD

**Issue by Nawdare** [squidfunk/mkdocs-material#5175]
### Translations

```Jinja
{% macro t(key) %}{{ {
  "language": "ku-IQ",
  "direction": "rtl",
  "action.edit": "دەستکاری ئەم پەڕە بکە",
  "action.skip": "ئەم ناوەڕۆکە بپەڕێنە",
  "action.view": "سەرچاوەی ئەم لاپەڕەیە نیشان بدە",
  "announce.dismiss": "دووبارە ئەمە پیشان مەدە",
  "clipboard.copy": "لەبەرگتنەوە بۆ کلیپبۆرد",
  "clipboard.copied": "لەبەرگیرایەوە بۆ کلیپ بۆرد",
  "consent.accept": "ڕازیبوون",
  "consent.manage": "بەڕیوەبردنی ڕیکخستنەکان",
  "consent.reject": "ڕەتکردنەوە",
  "footer": "ژێرپەڕە",
  "footer.next": "دواتر",
  "footer.previous": "پێشتر",
  "header": "ناونیشانی بەڕه",
  "meta.comments": "لێدوانەکان",
  "meta.source": "سەرجاوە",
  "nav": "ڕێنیشاندەر",
  "search": "گەڕان",
  "search.placeholder": "گەڕان",
  "search.share": "گەڕان",
  "search.reset": "سڕینەوە",
  "search.result.initializer": "ئامادەکردنی گەڕان",
  "search.result.placeholder": "بنووسە بۆ دەستپێکردن بە گەڕان",
  "search.result.none": "هیچ بەڵگەنامەیەکی هاوتا نیە",
  "search.result.one": "١ بەڵگەنامەی هاوتا",
  "search.result.other": "بەڵگەنامەی هاوتا #",
  "search.result.more.one": "١ دانەی تر لەسەر ئەم پەڕەیە",
  "search.result.more.other": "دانەی تر لەسەر ئەم پەڕەیە #",
  "search.result.term.missing": "ونبوو",
  "select.language": "زمان دیاریبکە",
  "select.version": "وەشان دیاریبکە",
  "source": "بڕۆ بۆ کۆگا",
  "source.file.date.created": "دروسکت کرا",
  "source.file.date.updated": "دوایین نوێکردنەوە",
  "tabs": "تابەکان",
  "toc": "خشتەی ناوەڕۆکەکان",
  "top": "گەڕانەوە بۆ سەرەوە"
}[key] }}{% endmacro %}
```

### Before submitting

- [X] I've checked the list of [available languages](https://squidfunk.github.io/mkdocs-material/setup/changing-the-language/#site-language) for existing translations.
- [X] I assure that the translations are accurate to my best knowledge.
- [ ] __Optional__: I want to integrate this translation myself and create a pull request following the [contribution guide](https://github.com/squidfunk/mkdocs-material/blob/master/CONTRIBUTING.md).

**Comment by jonaharagon** [comment]
The language code should probably be `ku-IQ` (as it's primarily used in Iraq) or `ku-Arab` (from [ISO 15924](https://en.wikipedia.org/wiki/ISO_15924#Arab), because it is written in the Sorani/Arabic alphabet) to differentiate it from the more common Northern Kurdish (which is written with the Latin alphabet so it could be `ku-Latn` or just `ku` if it's added in the future).

**Comment by squidfunk** [comment]
Thanks! Added in e60179f9f (and fixed the flag in cd73ef157 😅). Some translations are still missing, but it's great to have another language available!

**Comment by Nawdare** [comment]
yeah... your right @jonaharagon, it's not `ku`. but it shouldm'be also any of `ku-IQ` or `ku-Arab`. because [ISO](https://iso639-3.sil.org/code/kur) already has a code for it, and its `ckb` for the Central one. So... i will change it to it. 

(sorry for the poor research at first)

**Comment by jonaharagon** [comment]
ISO 639-3 can't be used, it has to be ISO 639-1, optionally with an ISO 3166-1 country and/or an ISO 15924 script variation. I think `ku-IQ` does make the most sense 👍 

https://en.m.wikipedia.org/wiki/Hreflang

**Comment by Nawdare** [comment]
np will go with that.

**Comment by squidfunk** [comment]
Released as part of [9.1.3](https://github.com/squidfunk/mkdocs-material/releases/tag/9.1.3).
